pom.xml

```diff
@@ -61,10 +62,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <configuration>
+                    <sourcepath>${project.build.directory}/generated-sources/delombok;${project.build.directory}/generated-sources/antlr4;${project.build.directory}/generated-sources/xjc</sourcepath>
```

Comment:
Additional config should not be provided by common lugin, antlr is very special and not used by many plugins

Reply:
You are right. I will remove it. C&P mistake.